src/rdkafka_cgrp.c

```diff
@@ -457,8 +457,11 @@ rd_kafka_cgrp_t *rd_kafka_cgrp_new(rd_kafka_t *rk,
         rkcg->rkcg_q                        = rd_kafka_consume_q_new(rk);
         rkcg->rkcg_group_instance_id =
             rd_kafkap_str_new(rk->rk_conf.group_instance_id, -1);
-        rkcg->rkcg_group_remote_assignor =
-            rd_kafkap_str_new(rk->rk_conf.group_remote_assignor, -1);
+
+        if (rkcg->rkcg_group_remote_assignor)
+                rkcg->rkcg_group_remote_assignor =
+                    rd_kafkap_str_new(rk->rk_conf.group_remote_assignor, -1);
+
```

Comment:
This change should be removed, here the `rkcg->rkcg_group_remote_assignor` is never set as it's been just allocated.